src/analyze/gui/mainwindow.cpp

```diff
@@ -404,6 +406,26 @@ MainWindow::MainWindow(QWidget* parent)
     m_ui->menu_File->addAction(m_closeAction);
     m_quitAction = KStandardAction::quit(qApp, SLOT(quit()), this);
     m_ui->menu_File->addAction(m_quitAction);
+    QShortcut* shortcut = new QShortcut(QKeySequence(QKeySequence::Copy), m_ui->stacksTree);
+    connect(shortcut, &QShortcut::activated,
+            [&]()
+    {
+      QTreeView* view = m_ui->stacksTree;
+      if (view->selectionModel()->hasSelection())
+      {
+          QString text;
+          QItemSelectionRange range = view->selectionModel()->selection().first();
```

Comment:
`const auto range = ...`

though you could also just loop over all ranges and append the texts if there are more than one (disjoint selection)